pkgs/development/python-modules/ttfautohint-py/default.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  gitUpdater,
+  pytestCheckHook,
+  setuptools,
+  setuptools-scm,
+  distutils,
+  ttfautohint,
+  fonttools,
+}:
+
+buildPythonPackage rec {
+  pname = "ttfautohint-py";
+  version = "0.5.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "fonttools";
+    repo = "ttfautohint-py";
+    rev = "v${version}";
```

Helps to avoid branch/tag confusion.

```suggestion
    rev = "refs/tags/v${version}";
```